src/Controls/tests/TestCases.HostApp/Issues/Issue22972.cs

```diff
@@ -0,0 +1,85 @@
+using Microsoft.Maui.Controls;
+
+namespace Maui.Controls.Sample.Issues
+{
+	[Issue(IssueTracker.Github, 22972, "Win platform WebView cannot be release after its parent window get close")]
+	public class Issue22972 : NavigationPage
+	{
+        ContentPage _rootPage = new ContentPage { Title = "Page 1" };
+		public Issue22972() 
+		{
+            PushAsync(_rootPage);
+            _rootPage.Content = new VerticalStackLayout()
+            {
+                new Label
+                {
+                    Text = "If you don't see a success label this test has failed"
+                }
+            };
+
+            _rootPage.Loaded += OnPageLoaded;
+        }
+
+		async void OnPageLoaded(object sender, EventArgs e)
+		{
+		    var references = new List<WeakReference>();
+            CurrentPage.Loaded -= OnPageLoaded;
+
+            {
+                var webView = new WebView
+                {
+                    HeightRequest = 500, // NOTE: non-zero size required for Windows
+                    Source = new HtmlWebViewSource { Html = "<p>hi</p>" },
+                };
+                var page = new ContentPage
+                {
+                    Content = new VerticalStackLayout { webView }
+                };
+                await Navigation.PushAsync(page);
+                await Task.Delay(1000); // give the WebView time to load
+
+                references.Add(new(webView));
+                references.Add(new(webView.Handler));
+                references.Add(new(webView.Handler.PlatformView));
+
+                await Navigation.PopAsync();
+            }
+
+
+            try
+            {
+                _rootPage.Content = new VerticalStackLayout()
+                {
+                    new Label
+                    {
+                        Text = "Waiting for resources to cleanup"
+                    }
+                };
+
+
+                // Assert *before* the Window is closed
+                await GarbageCollectionHelper.WaitForGC(references.ToArray());
+                _rootPage.Content = new VerticalStackLayout()
+                {
+                    new Label
+                    {
+                        Text = "Success, everything has been cleaned up",
+                        AutomationId = "Success"
+                    }
+                };
```

Comment:
This is just an idea to make it cleaner for future tests, but could you have methods like:
```csharp
LabelAssertionHelper.Pass(_rootPage);
// or if failed
LabelAssertionHelper.Fail(_rootPage, string.Join(", ", stillAlive));
```
Then you'd use these for every memory-related test that uses a `Label`.

Reply:
Alright, cleaned this one up to use an extension method

So, we can now more quickly add memory-related tests!